misc/docker-ci.mk

```diff
@@ -2,22 +2,23 @@ CONTAINER_NAME=h2oserver/h2o-ci:ubuntu2004
 SRC_DIR=/quicly
 CI_MK=$(SRC_DIR)/misc/docker-ci.mk
 CMAKE_ARGS=
+CMAKE_ENVS=
```

Comment:
Do we need to add a new way of passing arguments to CMake? I believe the compilers and compiler options can be passed as arguments to CMake, e.g., `-DCMAKE_C_COMPILER=clang`.